### PR TITLE
Animation issue

### PR DESCRIPTION
**Pull request by UncleFifi**
hacktoberfest-Pull-Request Making it so the box shakes if its empty and cant be changed

### CODE REVIEW

**Comment by UncleFifi** [comment]
I dont know how to resolve the conflict. they both look identical to me. would someone please tell me how I can resolve this on github.

**Comment by titivermeesch** [comment]
@UncleFifi You could pull the source branch and your IDE should show you the conflicts. It's not recommended to directly solve it on github.

**Comment by UncleFifi** [comment]
@titivermeesch thanks for support.
I believe I was able to resolve conflicts.
would this be okay

File: js/board.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ define(["js/properties", "js/sound", "js/squareImages", "js/info"], (properties,
     }
 
     function Board() {
-        this.squares = [];
+        this.squares = []
```
Comment (titivermeesch) [inline js/board.js:48]: I wasn't expecting a change here, if the ; got removed here this probably means that your eslint is not setup correctly.

File: js/board.js [hunk 2]
```diff
@@ -92,6 +92,7 @@ define(["js/properties", "js/sound", "js/squareImages", "js/info"], (properties,
                 squareImageElement.id = square.generateImageId();
                 squareImageElement.src = squareImages.generateImagePath(square.currentImage);
                 squareImageElement.draggable = false;
+                squareImageElement.setAttribute("isAnimating", "no")
```
Comment (titivermeesch) [inline js/board.js:95]: Semicolumn missing here

File: js/info.js [hunk 3]
```diff
@@ -41,19 +41,22 @@ define([], () => {
...
         generateInfo: function generateInfo(currentLevel, numberOfLevels) {
             levelNumber = `Level ${currentLevel}/${numberOfLevels}`;
             movements = 0;
             totalSeconds = 0;
-            let Element = document.createElement("div");
-            Element.className = "info";
-            Element.innerText = getInfo();
+            let infoWrapper = document.createElement("div");
+            infoWrapper.className = "info";
+            var infoChild = document.createElement("div")
```
Comment (titivermeesch) [inline js/info.js:54]: Please use const and let over var.

File: js/square.js [hunk 4]
```diff
@@ -1,5 +1,12 @@
 define(["js/properties", "js/squareImages", "js/sound"], (properties, squareImages, sound) => {
 
+    function animationCallBack()
```
Comment (titivermeesch) [inline js/square.js:3]: Is there a reason we don't use an arrow function here?

**Review by titivermeesch — CHANGES_REQUESTED** [review]
@UncleFifi Please see my comments

File: js/board.js (hunk 1 above)
Comment (UncleFifi) [reply js/board.js:48]: You're right I don't!
I have no idea why that is.
I'm not used to working with linting.

File: js/square.js (hunk 4 above)
Comment (UncleFifi) [reply js/square.js:3]: I think that was a merge conflict. Is that also a lint issue?

File: js/info.js (hunk 3 above)
Comment (UncleFifi) [reply js/info.js:54]: Will do

File: js/board.js (hunk 1 above)
Comment (titivermeesch) [reply js/board.js:48]: What IDE are you using? If it's VS code you can just install the eslint plugin and it should lint your code automatically.

File: js/square.js (hunk 4 above)
Comment (titivermeesch) [reply js/square.js:3]: No it's not really a linting issue. It's just that we try to use ES6 functions instead of using the old school functions.

**Comment by UncleFifi** [comment]
I have made the changes you have requested, please see #63 for changes

**Pull request merged**